.github/workflows/pytest.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           - python-version: "3.10"
             pytorch-version: nightly
             torchscript-version: 1.10.2
-            ray-version: 2.3.1
+            ray-version: nightly
```

Comment:
Should we do?

```
- ray-version: 2.3.1
- ray-version: 2.4.0
- ray-version: nightly
```

Reply:
I was planning to do that in a followup PR so we could encapsulate 2.4 fixes. How does that sound?